### PR TITLE
[vault] new command

### PR DESCRIPTION
**Pull request by vysahjk**

### CODE REVIEW

File: Babylon/commands/vault/get/user.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        return CommandResponse.fail()
+
+    data = response['data']
+    result = None
+    try:
+        result = data[name]
+    except Exception as exp:
+        logger.info(f"{exp} not found")
+        return CommandResponse.fail()
+    print(result)
```
Comment (enocquet) [inline Babylon/commands/vault/get/user.py:38]: forgotten 'print' ?

File: Babylon/commands/vault/login.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    ]
+
+    answers = inquirer.prompt(questions)
+    username = answers['username']
+    password = answers['password']
+    organization = answers['organization']
+    client = Client(url=env.server_id)
+    token = client.auth.userpass.login(username=username, password=password, mount_point=f"userpass-{organization}")
+    print(f"export BABYLON_TOKEN={token['auth']['client_token']}")
+    print(f"export BABYLON_ORG_NAME={organization}")
```
Comment (enocquet) [inline Babylon/commands/vault/login.py:32]: Print or logger ?

File: Babylon/commands/vault/get/get_global.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    """
+    Get a secret from global scope
+    """
+    check_alphanum(name)
+    schema = f"{env.organization_name}/{env.tenant_id}/global/{resource}/{name}"
+    response = hvac_client.read(path=schema)
+    if not response:
+        logger.info("Secret not found")
+        return CommandResponse.fail()
+    print(response['data']['secret'])
```
Comment (enocquet) [inline Babylon/commands/vault/get/get_global.py:33]: forgotten 'print' ?

File: Babylon/commands/vault/get/babylon.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    """
+    Get a secret from babylon scope
+    """
+    check_ascii(name)
+    schema = f"{env.organization_name}/{env.tenant_id}/babylon/{env.environ_id}/{name}"
+    response = hvac_client.read(path=schema)
+    if not response:
+        logger.info("Secret not found")
+        return CommandResponse.fail()
+    print(response['data']['secret'])
```
Comment (enocquet) [inline Babylon/commands/vault/get/babylon.py:31]: print ?

**Review by enocquet — CHANGES_REQUESTED** [review]

File: Babylon/commands/vault/get/user.py (hunk 1 above)
Comment (vysahjk) [reply Babylon/commands/vault/get/user.py:38]: i need print this in stdout for CI/CD reason (potentially). CommanResponse class returns a formatted string

File: Babylon/commands/vault/login.py (hunk 2 above)
Comment (vysahjk) [reply Babylon/commands/vault/login.py:32]: **print**: i need print this in stdout for users (they need copy-paste it)

**Review by enocquet — APPROVED** [review]

**Pull request merged**